angular.json

```diff
@@ -41,12 +41,12 @@
                 {
                   "type": "initial",
                   "maximumWarning": "500kb",
```

Comment:
I increased these budgets for the time being. Big things in our build include the data file and some of the scss files.